corehq/apps/dropbox/decorators.py

```diff
@@ -0,0 +1,25 @@
+from functools import wraps
+
+from django.http import HttpResponseRedirect
+
+from .utils import get_dropbox_auth_flow
+from .views import DROPBOX_ACCESS_TOKEN
+
+
+def require_dropbox_session(next_url=None):
```

Comment:
Any view that is wrapped with this will redirect to the dropbox authorization site if the user does not have an access_token in their session. this way we can safely assume in a view that we have the user's access token. next_url is just in case we do get redirected, we can define where we go next